rllib/utils/tests/test_actor_manager.py

```diff
@@ -32,9 +33,7 @@ def __init__(self, i, maybe_crash=True):
         self.random_numbers = RANDOM_NUMS[i]
         self.count = 0
         self.maybe_crash = maybe_crash
-        self.config = {
-            "recreate_failed_workers": True,
-        }
+        self.config = AlgorithmConfig().rollouts(recreate_failed_workers=True)
```

Comment:
Not sure why, but some tests also failed inside `FaultAwareApply`, which will acces self.config. recreate_failed_workers, which will not work for a dict.

Reply:
This line makes it so that self.config.recreate_failed_workers itself does not fail.